test/e2e/install/install_test.go

```diff
@@ -90,12 +89,45 @@ var _ = Describe("[Install] continuousIntegration", func() {
 			}, 5*time.Minute, 10*time.Second).Should(BeTrue(), "RTE condition did not become available")
 
 			By("checking the NRT CRD is deployed")
-			crd := &apiextensionv1.CustomResourceDefinition{}
-			key := client.ObjectKey{
-				Name: crdName,
-			}
-			err := e2eclient.Client.Get(context.TODO(), key, crd)
+			_, err := crds.GetByName(e2eclient.Client, crds.CrdNRTName)
 			Expect(err).NotTo(HaveOccurred())
+
+			By("checking the NRO CRD is deployed")
+			_, err = crds.GetByName(e2eclient.Client, crds.CrdNROName)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("checking Daemonset is up&running")
+			const DSCheckTimeout = 1 * time.Minute
+			const DSCheckPollingPeriod = 5 * time.Second
+			Eventually(func() bool {
+
+				ds, err := getDaemonSetByOwnerReference(updatedNROObj.UID)
+				if err != nil {
+					klog.Warningf("unable to get Daemonset  %v", err)
+					return false
+				}
+
+				if ds.Status.NumberMisscheduled != 0 {
+					klog.Warningf(" Misscheduled: There are %d nodes that should not be running Daemon pod but are", ds.Status.NumberMisscheduled)
+					return false
+				}
+
+				if ds.Status.NumberUnavailable != 0 {
+					klog.Infof(" NumberUnavailable %d (should be 0)", ds.Status.NumberUnavailable)
+					return false
+				}
+
+				if ds.Status.CurrentNumberScheduled != ds.Status.DesiredNumberScheduled {
+					klog.Infof(" CurrentNumberScheduled %d (should be %d)", ds.Status.CurrentNumberScheduled, ds.Status.DesiredNumberScheduled)
+					return false
+				}
+
+				if ds.Status.NumberReady != ds.Status.DesiredNumberScheduled {
+					klog.Infof(" NumberReady %d (should be %d)", ds.Status.CurrentNumberScheduled, ds.Status.DesiredNumberScheduled)
+					return false
+				}
+				return true
+			}, DSCheckTimeout, DSCheckPollingPeriod).Should(BeTrue(), "DaemonSet Status was not correct")
```

Comment:
for the next time: "DaemonSet status is not correct" is too generic. The test should clearly signal at glance what the expectation is and what the actual state is. Yes, we (correctly) do logging, but in some test runners we may want not to enable the logs by default, so better to reiterate the failed condition as precisely as possible.